src/main/java/seedu/address/logic/parser/UnmatchCommandParser.java

```diff
@@ -0,0 +1,42 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.UnmatchCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a UnmatchCommand object
+ */
+public class UnmatchCommandParser implements Parser<UnmatchCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the UnmatchCommand
+     * and returns a UnmatchCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
```

Comment:
Same issue as above, for more information you can read my text in Telegram regarding the JavaDoc style.